README.md

```diff
@@ -1,4 +1,20 @@
 # TFExplorer
+[![Build Status](https://travis-ci.org/hardlyhuman/TFExplorer.svg?branch=master)](https://travis-ci.org/hardlyhuman/TFExplorer)
+
+
+<b>TFExplorer enables developers to quickly find out the relevant TensorFlow API's for their needs & requirements.
```

Comment:
Please don't use HTML tags in markdown. Markdown has features for bold italic etc
You can find a markdown cheatsheet here: [Markdown Cheatsheet](https://duckduckgo.com/?q=markdown+cheat+sheet&ia=answer&iax=answer)

Reply:
Okay, thanks for informing!